torch/utils/collect_env.py

```diff
@@ -424,6 +425,17 @@ def main():
     output = get_pretty_env_info()
     print(output)
 
+    minidump_dir = torch.utils._crash_handler.DEFAULT_MINIDUMP_DIR
+    if sys.platform == "linux" and os.path.exists(minidump_dir):
+        dumps = [os.path.join(minidump_dir, dump) for dump in os.listdir(minidump_dir)]
+        latest = max(dumps, key=os.path.getctime)
+        ctime = os.path.getctime(latest)
+        creation_time = datetime.datetime.fromtimestamp(ctime).strftime('%Y-%m-%d %H:%M:%S')
+        print(f"\n *** Detected a minidump at {latest} created on {creation_time}, "
+              "if this is related to your bug please include it when you file a report ***",
+              file=sys.stderr)
```

Comment:
`collect_env` tool is weird in a sense that latest version of the tool should work with say PyTorch-1.4, which would not have this symbol defined.  Also, not sure if this is true, but if it must be compatible with Python-2, please avoid using f-strings

Reply:
There were some f-strings in there already, but I cleaned all that up and it works under Python 2.7 again